README.md

```diff
@@ -0,0 +1,42 @@
+## omsoo-portfolio
+
+The project is still in the works. Constructive feedback is encouraged. Just completed the structure for the destop version and mobile view.
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly deploy your project on GitHub pages and add the demo link to your README.